nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ParseCEF.java

```diff
@@ -339,7 +351,7 @@ public void process(OutputStream out) throws IOException {
             session.transfer(flowFile, REL_SUCCESS);
         } catch (CEFHandlingException e) {
             // The flowfile has failed parsing & validation, routing to failure and committing
-            getLogger().error("Failed to parse {} as a CEF message due to {}; routing to failure", new Object[] {flowFile, e});
+            getLogger().error("Failed to parse {} as a CEF message due to {}; routing to failure", new Object[] {flowFile, e}, e);
```

Comment:
The error logging could also be adjusted to avoid repeating the exception:
```suggestion
            getLogger().error("Reading CEF Event Failed: {}", flowFile, e);
```